ouroboros-consensus/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -774,19 +863,25 @@ mempoolCapTestExpectedTrace mcts = go chunks
 -- | A data type containing 'StrictTVar's by which the two threads spawned by
 -- 'prop_Mempool_Capacity' can coordinate with each other.
 data MempoolCapTestEnv m = MempoolCapTestEnv
-  { mctEnvAddedTxs   :: StrictTVar m Word
-    -- ^ The number of transactions which have been added to the mempool.
-  , mctEnvRemovedTxs :: StrictTVar m Word
+  { mctEnvAddedTxs             :: StrictTVar m Word32
+    -- ^ The total number of transactions which have been added to the
+    -- mempool.
+  , mctEnvRemovedTxs           :: StrictTVar m Word
```

Comment:
This should be the same type as `mctEnvAddedTxs`. I suggest you change them both to `Int` as it is just a count used in the tests, so it'll get rid of some `fromIntegral`s.